packages/client/src/store/CallState.ts

```diff
@@ -1299,4 +1393,50 @@ export class CallState {
       this.setCurrentValue(this.ownCapabilitiesSubject, event.own_capabilities);
     }
   };
+
+  private updateClosedCaptions = (event: ClosedCaptionEvent) => {
```

Comment:
Suggestion: some event handlers we keep outside of the state class, in `src/events`. Since CallState is getting rather big, should we consider extracting this handler there as well?

Reply:
It used to be like that in the past. We have moved to a single class-does-it-all approach to be consistent with the other SDKs. So, at this moment, I wouldn't be breaking this class into pieces :)